api/lib/project-fields-nodes-to-fields-map.js

```diff
@@ -129,6 +129,28 @@ export function projectFieldsNodesToFieldsMap(state, project, nodes) {
         );
       }
 
+      // If the field is of type "Iteration", then the `configuration` property will be set.
+      if (node.configuration) {
```

Comment:
I wonder if we maybe should check for `node.configuration?.iterations` to be truthy? Just in case the `.configuration` will be used by other fields in future?